src/firebase/auth_ui.js

```diff
@@ -0,0 +1,18 @@
+import firebase from "firebase/compat/app";
+import "firebase/compat/auth";
+import * as firebaseui from "firebaseui";
+import auth from "./auth";
+
+export const authUiConfig = {
+  signInSuccessUrl: "/",
+  signInOptions: [
+    {
+      provider: firebase.auth.EmailAuthProvider.PROVIDER_ID,
+      customParameters: {
+        prompt: "select_account"
+      }
+    }
+  ]
+};
+
+export default new firebaseui.auth.AuthUI(auth);
```

Comment:
Puts all our Auth UI imports and config in one place, and exports a function we can use elsewhere, and a variable we can use elsewhere

I'm not super clear on what the function days. Might be clearer when I see it used